onnxruntime/test/testdata/custom_op_library/custom_op_library.cc

```diff
@@ -73,6 +79,45 @@ struct KernelOne {
   Ort::CustomOpApi ort_;
 };
 
+struct KernelOneCuda {
+  KernelOneCuda(OrtApi api)
```

Comment:
Any reason we need to add a new custom kernel ? Can we just add the #ifdef in KernelOne ? KernelOneCuda is essentially KernelOne + the ifdef : it seems like code is unnecessarily duplicated in this file.